Create ciclo_de_vida

messiasgomes540 opened the pull request.

messiasgomes540 committed:
Create ciclo_de_vida

messiasgomes540 committed:
Create Fundamentos_de_Projetos

messiasgomes540 committed:
Create 05-PMBOK.md

em andamento

messiasgomes540 committed:
Create 06-prince2.md

Resumo atividade 06-prince2

messiasgomes540 committed:
Create Tempo-gasto-com-celular.md

atividade tempo gasto com celular semanal

messiasgomes540 committed:
Create  08-gerente.md

atividade 08-gerente

messiasgomes540 committed:
Update 08-gerente.md

atualizando a tarefa

messiasgomes540 committed:
Create 09-gerente.md

ativiadde 09-gerente

messiasgomes540 committed:
Create 10-lider.md

ativiadade lider-1

messiasgomes540 committed:
Create 11-lider.md

tarefa extar lider 11

leonardo-minora commented:
Tu tens que criar um diretório com teu nome e colocar os arquivos lá.
Além disso, precisa colocar cabeçalhos nos teus trabalhos.

No aguardo... @messiasgomes540

messiasgomes540 committed:
craida pasta com meu nome para atividades

messiasgomes540 committed:
Create 02-projetos.md

add  2 - Conceitos Fundamentais de Projetos

messiasgomes540 committed:
Add 04_ciclo-de-vida

Ciclo de Vida de um Projeto: o que é e quais as suas principais características

messiasgomes540 committed:
Add 05-pmbok.md

PMBOK

messiasgomes540 committed:
Add 06-prince2.md

06 - Fundamentos do UK PRINCE2

messiasgomes540 committed:
Add 08-gerente.md

Gerente Tarefa 1

messiasgomes540 committed:
Add 09-gerente.md

Gerente Tarefa 2

messiasgomes540 committed:
Add 10-Lider.md

Líder Tarefa 1

messiasgomes540 committed:
Add 11-lider.md

Líder Tarefa  extra

messiasgomes540 committed:
Rename 05-pmbok.md to movido-05-pmbok.md

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Update and rename ciclo_de_vida to Messias-gomes/movido-04-ciclo-de-vida.md

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Rename 06-prince2.md to Messias-gomes/movido-06-prince2.md

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Rename 08-gerente.md to Messias-gomes/movido-08-gerente.md

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Rename 09-gerente.md to Messias-gomes/movido-09-gerente.md

movendo arquivos para a pasta com meu nome

messiasgomes540 committed:
Rename Messias-gomes to Messias-gomes/movido-10-lider.md

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Rename 11-lider.md to Messias-gomes/movido-11-lider.md

messiasgomes540 committed:
Rename Fundamentos_de_Projetos to Messias-gomes/movido-Fundamentos_de_Projetos

movendo arquivos para pasta com meu nome

messiasgomes540 committed:
Rename Tempo-gasto-com-celular.md to Messias-gomes/movido-Tempo-gasto-com-celular.md

movendo arquivos para pasta com meu nome
